fala/locale/cy/LC_MESSAGES/django.po

```diff
@@ -92,6 +92,14 @@ msgstr ""
 "oes gennych blentyn sydd wedi cael anaf i’r ymennydd yn ystod beichiogrwydd, "
 "genedigaeth neu yn ystod 8 wythnos gyntaf ei fywyd."
 
+#: fala/common/states.py:71
```

Comment:
does it matter that these numbers are out of order? e.g.  Is there any benefit while debugging/ trying to find messages where these ids are used and we would look for them? If not no worries, if they contribute in any meaningful way should these be moved to a position that matches the numerical value?

Reply:
@willc-work So these paths & numbers refer to the document the text is in and the line which the text is placed on. i.e  for this `fala/common/states.py:71`, `states.py` file and line number 71.

In terms of order this is an automatic process, which lists them alphabetically (file path) then numerically (line within the file).

I think that is what you were asking. Sorry if I've misunderstood.